test/8-data-benchmarks.js

```diff
@@ -12,7 +12,10 @@ describe('8 - does some benchmarks on api calls, data events and events', functi
 
   this.timeout(120000);
 
-  var maximumPings = 1000;
+  // var maximumPings = 1000;
+  var maximumPings = 900; // with local exchange methods calling directly instead of through happn this
```

Comment:
Oh, I see it is not actually on callback, but calling directly. Sacrificing some speed perhaps we should do a setImmediate  before calling operate. This will make sure we don't have call stack issues and also just create a better behaved system.